client/src/Clusters/Footer/Footer.style.css

```diff
@@ -0,0 +1,25 @@
+.footer-section {
+    border-top: 1px solid #7f7f7f;
+    margin: auto;
+    padding-top: 30px;
+    width: 90%;
+}
+
+.footer-left {
+    float: left;
+}
+
+.footer-left h2 {
+    color:#7f7f7f;
+    padding-left: 30px;
```

Comment:
it may seem like margin and paddings are interchangeable sometimes, but they have different behaviors. i.e. if this element has a hover effect and or is clicking, when using a padding, that space will be clickable, too.